metals/src/main/scala/scala/meta/internal/metals/Command.scala

```diff
@@ -18,6 +18,12 @@ case class Command(
 }
 
 object Argument {
+  def getParamsAsString(
```

Comment:
You can now change this one to use the default command parsing as https://github.com/scalameta/metals/pull/3149#pullrequestreview-761777873 is merged.